core/frontend/src/internal/tile/RealityModelTileTree.ts

```diff
@@ -979,9 +1001,48 @@ export class RealityTreeReference extends RealityModelTileTree.Reference {
     }
   }
 
-  public override async addAttributions(cards: HTMLTableElement): Promise<void> {
-    // eslint-disable-next-line @typescript-eslint/no-deprecated
-    return Promise.resolve(this.addLogoCards(cards));
+  public override async addAttributions(cards: HTMLTableElement, vp: ScreenViewport): Promise<void> {
+    const tiles = IModelApp.tileAdmin.getTilesForUser(vp)?.selected;
+    const copyrightMap = new Map<string, number>();
+    if (tiles) {
+      for (const tile of tiles as Set<RealityTile>) {
+        if (tile.copyright) {
+          for (const copyright of tile.copyright?.split(";")) {
+            const currentCount = copyrightMap.get(copyright);
+            copyrightMap.set(copyright, currentCount ? currentCount + 1 : 1);
+          }
+        }
+      }
+    }
+
+    // Only add another logo card if the tiles have copyright
+    if (copyrightMap.size > 0) {
+      // Order by most occurances to least
+      // See https://developers.google.com/maps/documentation/tile/create-renderer#display-attributions

```

Comment:
based off of this, i wonder if we need an additional / clickable `data attributions` thats always visible on the vp and when clicked also shows the logo card

Reply:
For reference; more details about attribution requirements are discussed here: https://github.com/iTwin/itwinjs-core/issues/8036#issuecomment-2895776072